### PR TITLE
Added tests for GitHub getPR function.

### PR DESCRIPTION
**Pull request by janetchen1**

### CODE REVIEW

**Comment by waisbrot** [comment]
Though it's passing the new tests, the code appears incomplete. Could you make a note in the PR when it's incomplete?

**Comment by janetchen1** [comment]
That's my bad-- I had written tests but I guess I didn't stash them correctly when I merged with master. Will add those back in.

File: github.js [hunk 1]
```diff
@@ -1,6 +1,43 @@
...
+//                         "channel": channel,
+//                         "text": prs + " open pull request(s) in " + repo.name + " " +
+//                         repo.html_url
+//                     });
+//                 }
+
+//             });
+//         });
+//     });
+// }
```
Comment (waisbrot) [inline github.js:39]: Let's remove this. It's best to avoid checking commented-out code in.

**Comment by waisbrot** [comment]
Everything in here was also in #18, I think.  If that's true, please delete the branch. If not, let's fix it up. :-)

**Pull request closed without merging**